examples/farmer/farmer_cylinders.py

```diff
@@ -117,6 +117,10 @@ def main():
 
     if write_solution:
         sputils.write_spin_the_wheel_first_stage_solution(spcomm, opt_dict, 'farmer_plant.csv')
+        sputils.write_spin_the_wheel_first_stage_solution(spcomm,
+                                                          opt_dict,
+                                                          'farmer_cyl_nonants.spy',
+                                                          first_stage_solution_writer=sputils.first_stage_nonant_npy)
```

Comment:
We should probably rename the kwarg `first_stage_solution_writer` to `first_stage_nonant_writer` for this function -- and maybe the function itself should be renamed `write_spin_the_wheel_nonant_solution`.